packages/fiori/src/SideNavigation.ts

```diff
@@ -7,50 +7,47 @@ import property from "@ui5/webcomponents-base/dist/decorators/property.js";
 import slot from "@ui5/webcomponents-base/dist/decorators/slot.js";
 import { getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
 import type I18nBundle from "@ui5/webcomponents-base/dist/i18nBundle.js";
-import List from "@ui5/webcomponents/dist/List.js";
-import StandardListItem from "@ui5/webcomponents/dist/StandardListItem.js";
-import Tree from "@ui5/webcomponents/dist/Tree.js";
-import TreeItem from "@ui5/webcomponents/dist/TreeItem.js";
-import type { TreeItemClickEventDetail } from "@ui5/webcomponents/dist/Tree.js";
+import ItemNavigation from "@ui5/webcomponents-base/dist/delegate/ItemNavigation.js";
+import type { ITabbable } from "@ui5/webcomponents-base/dist/delegate/ItemNavigation.js";
+import {
+	isPhone,
+	isTablet,
+	isCombi,
+} from "@ui5/webcomponents-base/dist/Device.js";
+import NavigationMode from "@ui5/webcomponents-base/dist/types/NavigationMode.js";
+import SideNavigationItemBase from "./SideNavigationItemBase.js";
```

Comment:
You are using it only as a a typed parameter/return parameter, so import it only as a type.